skore/src/skore/sklearn/_plot/data/table_report.py

```diff
@@ -0,0 +1,49 @@
+from skrub._reporting._html import to_html
+from skrub._reporting._summarize import summarize_dataframe
+
+from skore.sklearn._plot.base import Display
+
+
+class TableReportDisplay(Display):
+    def __init__(self, summary, column_filters=None):
+        self.summary = summary
+        self.column_filters = column_filters
+
+    @classmethod
+    def from_frame(cls, df, with_plots=True, title=None):
+        summary = summarize_dataframe(
+            df,
+            with_plots=with_plots,
+            title=title,
+        )
+        return cls(summary)
+
+    def plot(self, kind=None):
+        if kind is None:
+            return self.html_snippet()
+        # TODO
+        raise NotImplementedError()
+
+    def frame(self):
+        return self.summary
+
+    def html_snippet(self):
+        """Get the report as an HTML fragment that can be inserted in a page.
+
+        Returns
+        -------
+        str :
+            The HTML snippet.
+        """
+        return to_html(
+            self.summary,
+            standalone=False,
+            column_filters=self.column_filters,
+        )
+
+    def _repr_mimebundle_(self, include=None, exclude=None):
```

Comment:
Let's do a mixin for those like in scikit-learn: https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/utils/_repr_html/base.py#L118-L152

We are going to reuse it many time I think for the different display.